app/assets/stylesheets/screens/_keywords.scss

```diff
@@ -0,0 +1,17 @@
+.keywords-table {
+  @extend .border;
```

Comment:
Avoid over-usage of `@extend` as with SCSS it results in mega-selectors. That is why its usage is limited: https://nimblehq.co/compass/development/code-conventions/css/#convention-52

Reply:
Fixed in 2c01246cc398ebb22611ccf7767f104820deb34f